docs/hub-tutorials/join-mainnet.md

```diff
@@ -120,8 +120,7 @@ gaiad start
 
 Your node needs to know how to find peers. You'll need to add healthy seed nodes to `$HOME/.gaia/config/config.toml`. The [`launch`](https://github.com/cosmos/launch) repo contains links to some seed nodes.
 
-If those seeds aren't working, you can find more seeds and persistent peers on a Cosmos Hub explorer (a list can be found on the [launch page](https://cosmos.network/launch)).
-
+If those seeds aren't working, you can find more seeds and persistent peers on a Cosmos Hub explorer (try [mintscan](https://www.mintscan.io/cosmos) or [bigdipper](https://cosmos.bigdipper.live/)). The Hub team is currently working on a forthcoming guide with additional detail.
```

Comment:
You can't actually find seed nodes on the explorers unfortunately. But quicksync has an address book you can download. @okwme are we OK to refer people to that? Centralisation and all that.

```suggestion
If those seeds aren't working, you can download an address book with more seeds and persistent peers on [QuickSync](https://quicksync.io/networks/cosmos.html) by ChainLayer. Please note that this is an external platform and that we encourage due diligence when adding untrusted peers. The Hub team is currently working on a forthcoming guide with additional details.
```

Reply:
> You can't actually find seed nodes on the explorers unfortunately. But quicksync has an address book you can download. @okwme are we OK to refer people to that? Centralisation and all that.

Not sure how well it's maintained, but there are seeds here too! https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json

Reply:
Ah yeah, that's a great one actually, forgot that, thanks Udit! I think we can go ahead and use those as well, the registry is being actively maintained and will hopefully be the main source for this seeds in the future.

```suggestion
If those seeds aren't working, you can use the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json) to find more. Alternatively, you can download an address book with more seeds and persistent peers on [QuickSync](https://quicksync.io/networks/cosmos.html) by ChainLayer. Please note that this is an external platform and that we encourage due diligence when adding untrusted peers. The Hub team is currently working on a forthcoming guide with additional details.```